src/IRM/lib/ExpLib.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2023 Morpho Association
+pragma solidity ^0.8.0;
+
+/// @title ExpLib
+/// @custom:contact security@euler.xyz
+/// @author Euler Labs (https://www.eulerlabs.com/).
```

Comment:
I don't think we should claim authorship over this file: We didn't make any significant changes.

Reply:
Changed it, how is it now?

Reply:
LGTM